src/defaults.js

```diff
@@ -88,9 +88,9 @@ export default {
      */
     container: [
       '<div class="datepicker__container">',
-        '<%= renderHeader() %>',
-        '<%= renderTimepicker() %>',
-        '<%= renderCalendar() %>',
+        '<%= obj.renderHeader() %>',
```

Comment:
The templating syntax is slightly uglier now, but doesn't require `with` any more because its deprecated.